src/fileops/davmeta.hpp

```diff
@@ -115,6 +115,31 @@ class AzureMetaOps : public HttpIOChain{
 
 };
 
+/// The HttpMetaOps handle all meta-data operations related to Swift
```

Comment:
I believe the "HttpMetaOps" is a refactoring remnant from when HttpMetaOps was handling S3, before it was moved into S3MetaOps. 

Follow the style of the AzureMetaOps, which was more recently added.